Makefile

```diff
@@ -47,7 +47,7 @@ setup-mysql: drop-mysql
 		TO 'orator'@'localhost';"
 
 drop-mysql:
-	@type -p psql > /dev/null || { echo 'Install and setup PostgreSQL'; exit 1; }
+	@type -p psql > /dev/null || { echo 'Install and setup MySQL'; exit 1; }
```

Comment:
why does it say `@type -p psql`? Maybe it is checking for postgres here?

Reply:
I should change that too `mysql`.